spyrmsd/rmsd.py

```diff
@@ -373,3 +373,177 @@ def rmsdwrapper(
             )
 
     return RMSDlist
+
+
+import os
+from concurrent.futures import ProcessPoolExecutor
+from functools import partial
+from multiprocessing import Process, Queue
```

Comment:
You can also move this up so that CI becomes less noisy. GitHub is pretty good at hiding all that is untouched and only showing what really matters (so we would clearly see this even at the top of the file).

Reply:
You're right, I'll keep it in mind next time. I already fixed it in the latest commit!